packages/kit/test/utils.js

```diff
@@ -112,7 +112,8 @@ export const test = base.extend({
 		/** @param {string} selector */
 		async function in_view(selector) {
 			const box = await page.locator(selector).boundingBox();
-			return !!box;
+			const view = await page.viewportSize();
+			return box && view && box.y < view.height && box.y + box.height > 0;
```

Comment:
Is this part right `box.y + box.height > 0`? Or should it be `-` instead of `+`?

Reply:
Looks right to me. The `box.y` is the top value of the box, plus the height and comparing it with 0 ensures that at least a part of the bounding box is inside the viewport. For example, cases where `box.y` is negative (slightly outside of the viewport), but its height is still visible in the viewport.

Reply:
oh, I see. yeah, makes sense